x-pack/test/security_solution_cypress/runner.ts

```diff
@@ -125,3 +125,34 @@ export async function SecuritySolutionCypressVisualTestRunner({ getService }: Ft
     });
   });
 }
+
+export async function SecuritySolutionCypressUpgradeCliTestRunner({
+  getService,
+}: FtrProviderContext) {
+  const log = getService('log');
+
+  await withProcRunner(log, async (procs) => {
+    await procs.run('cypress', {
+      cmd: 'yarn',
+      args: ['cypress:run:upgrade'],
+      cwd: resolve(__dirname, '../../plugins/security_solution'),
+      env: {
+        FORCE_COLOR: '1',
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        CYPRESS_baseUrl: process.env.TEST_KIBANA_URL,
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        CYPRESS_protocol: process.env.TEST_KIBANA_PROTOCOL,
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        CYPRESS_hostname: process.env.TEST_KIBANA_HOSTNAME,
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        CYPRESS_configport: process.env.TEST_KIBANA_PORT,
+        CYPRESS_ELASTICSEARCH_URL: process.env.TEST_ELASTICSEARCH_URL,
+        CYPRESS_ELASTICSEARCH_USERNAME: process.env.TEST_ES_USER,
+        CYPRESS_ELASTICSEARCH_PASSWORD: process.env.TESTES_PASSWORD,
```

Comment:
Should this be ```process.env.TEST_ES_PASSWORD``` ?

Reply:
Totally! I don't know what is happening to me lately that I'm doing typos everywhere

Reply:
Actually I think it is ```TEST_ES_PASS```